src/BlazorWasmSdk/Tasks/ComputeBlazorBuildAssets.cs

```diff
@@ -118,6 +118,28 @@ public override bool Execute()
                         assetCandidates.Add(newDotNetJs);
                         continue;
                     }
+                    else if(candidate.GetMetadata("FileName") == "dotnet-crypto-worker" && candidate.GetMetadata("Extension") == ".js")
```

Comment:
```suggestion
                    else if (candidate.GetMetadata("FileName") == "dotnet-crypto-worker" && candidate.GetMetadata("Extension") == ".js")
```

Also are these definitely case-insensitive?